src/test/data/JsonSerializableAddressBookTest/duplicatePersonAddressBook.json

```diff
@@ -1,14 +1,16 @@
 {
   "persons": [ {
     "name": "Alice Pauline",
-    "phone": "94351253",
+    "parent phone one": "94824242",
+    "parent phone two": "91204323",
     "email": "alice@example.com",
     "address": "123, Jurong West Ave 6, #08-111",
     "studentId": "12345",
     "tags": [ "friends" ]
   }, {
     "name": "Bob ddd",
-    "phone": "94351253",
+    "parent phone one": "94824242",
+    "parent phone two": "91204323",
```

Comment:
This part actually only need studentId for the student to be the same, but it is ok to have duplicated phone number.